shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/consumercart/ConsumerCartViewModel.kt

```diff
@@ -125,13 +133,39 @@ class ConsumerCartViewModel(
         )
     }
 
-    private fun onProductClicked(uuid: String, name: String) {
+    private fun onProductClicked(
+        uuid: String,
+        name: String,
+        productDetailsOpenedFrom: ProductDetailsOpenedFrom,
+    ) {
         event {
-            ConsumerCart.Event.NavigateToProduct(uuid = uuid, name = name)
+            ConsumerCart.Event.NavigateToProduct(
+                uuid = uuid,
+                name = name,
+                productDetailsOpenedFrom = productDetailsOpenedFrom
+            )
         }
     }
 
-    private fun onAddCardProductClicked(menuProductUuid: String) {
+    private fun addRecommendationProductClicked(menuProductUuid: String) {
+        analyticService.sendEvent(
+            RecommendationAddEvent,
+            params = listOf(EventParameter("menuProductUuid", menuProductUuid)),
+        )
+
+        addProduct(
+            menuProductUuid = menuProductUuid
+        )
+    }
+
+    private fun addCartProductToCartClick(menuProductUuid: String) {
+        addProduct(
+            menuProductUuid = menuProductUuid
+        )
+        analyticService.sendEvent(CartAddEvent)
```

Comment:
- Let's rename this event to `AddCartProductToCartEvent`, this is really confusing now
- Why do we send `Add` event but don't send `Remove` event?